lib/charms/kratos/v0/kratos_endpoints.py

```diff
@@ -94,13 +93,14 @@ def send_endpoint_relation_data(self, admin_endpoint: str, public_endpoint: str)
             return
 
         relations = self.model.relations[self._relation_name]
+        endpoints_databag = {
+            "admin_endpoint": admin_endpoint,
+            "public_endpoint": public_endpoint,
+            "login_browser_endpoint": f"{public_endpoint}/self-service/login/browser",
+            "sessions_endpoint": f"{public_endpoint}/sessions/whoami",
```

Comment:
We need to be consistent, we either provide the base URL  and it is the other party's responsibility to construct the full path (current implementation) or we provide all the different endpoints (your change). I am not against this approach, but we need to follow only one.

Reply:
I think we follow the latter (see [login ui lib](https://github.com/canonical/identity-platform-login-ui-operator/blob/main/lib/charms/identity_platform_login_ui_operator/v0/login_ui_endpoints.py#L111)). I don't think the other party should construct the path, it's easier to manage in the charm that provides the endpoint (in case the api ever changes).